tests/e2e/test_add_endpoints_and_lookups.py

```diff
@@ -248,60 +248,6 @@ def test_command_add_endpoints_and_lookups_success_lookups_required(
         assert endpoint.get("entry-date", None) == expected_entry_date
 
 
-def test_command_add_endpoints_and_lookups_failure_incorrect_dataset(
```

Comment:
Yes right to remove it, but I'm wondering with the above removal of an error and this test is there any other errors that can occur e.g. range max has been hit or maybe a warning that states the current is below the minimum?

Reply:
They may already be included